File: unity/Assets/Hallway/Constructor/RPG/tableJump.js
#pragma strict

var Bootstrap : GameObject;

function OnTriggerEnter() {
	var scr1 : RPGParser = Bootstrap.GetComponent.<RPGParser>();
	if (!scr1.RPG.tableJump) {
		scr1.Achievement("Прыжок на стол!\n+10 очков!", 10);
		scr1.RPG.tableJump = true;
	}
}

File: unity/Assets/Hallway/Constructor/RPG/tableJump.cs
using UnityEngine;

public class tableJump : MonoBehaviour
{
    public GameObject Bootstrap;

    void OnTriggerEnter()
    {
        RPGParser rpgParser = Bootstrap.GetComponent<RPGParser>();
        if (!rpgParser.RPG.tableJump)
        {
            rpgParser.Achievement("Прыжок на стол!\n+10 очков!", 10);
            rpgParser.RPG.tableJump = true;
        }
    }
}
